### PR TITLE
feat: add swift implementation to lcof2 problem: No.057

### PR DESCRIPTION
**Pull request by klever34**

### CODE REVIEW

File: lcof2/剑指 Offer II 057. 值和下标之差都在给定的范围内/README.md [hunk 1]
```diff
@@ -809,6 +809,53 @@ class TreeMultiSet<T = number> {
...
+    func ceiling(_ value: Element) -> Element? {
+        for element in elements {
+            if element >= value {
+                return element
+            }
+        }
+        return nil
+    }
+}
+```
```
Comment (yanglbme) [inline lcof2/剑指 Offer II 057. 值和下标之差都在给定的范围内/README.md:857]: @klever34 

Hello, the SortedSet implemented in this way has low efficiency and cannot pass all test cases within the time limit. It needs to be reimplemented.

**Review by yanglbme — CHANGES_REQUESTED** [review]

File: lcof2/剑指 Offer II 057. 值和下标之差都在给定的范围内/README.md (hunk 1 above)
Comment (yanglbme) [reply lcof2/剑指 Offer II 057. 值和下标之差都在给定的范围内/README.md:857]: https://leetcode.com/problems/contains-duplicate-iii/

Comment (klever34) [reply lcof2/剑指 Offer II 057. 值和下标之差都在给定的范围内/README.md]: Thanks. It has been taken care of. Kindly review again.

Comment (yanglbme) [reply lcof2/剑指 Offer II 057. 值和下标之差都在给定的范围内/README.md]: image

:(

Comment (klever34) [reply lcof2/剑指 Offer II 057. 值和下标之差都在给定的范围内/README.md]: Interesting, let me take a look at it.

**Comment by klever34** [comment]
@yanglbme Did you push a fix for this?

**Comment by yanglbme** [comment]
> @yanglbme Did you push a fix for this?

@klever34 Yes, I have added a TreeSet template class implemented with AVL.

**Review by yanglbme — APPROVED** [review]

**Pull request merged**